### PR TITLE
Turn keiko reports directory into a Docker volume

### PR DESCRIPTION
**Pull request by Donnype**
## Changes
Turn keiko reports into a volume and refactor towards one settings instance instead of several global variables. This would resolve permission issues we are having at times for Keiko, such as when users are running Docker in rootless mode.

### CODE REVIEW

**Review by Lisser — APPROVED** [review]
Top!

File: keiko/Makefile [hunk 1]
```diff
@@ -34,7 +34,7 @@ test:
 	python -m unittest discover -s tests -p "test_*.py"
 
 itest:
-	docker-compose -f docker-compose-base.yml -f .ci/docker-compose.yml up -d --build
+	docker-compose -f docker-compose-base.yml -f .ci/docker-compose.yml up -d --build && sleep 1
```
[the remark this answers is not in the document]
Comment (Donnype) [reply keiko/Makefile:37]: Indeed timing issue: couldn't get it to work without it..

[the remark this answers is not in the document]
Comment (Donnype) [reply keiko/Makefile:37]: https://github.com/minvws/nl-kat-coordination/issues/459

**Comment by dekkers** [comment]
Also looks good to me, so I put the PR on the project board in QA review / testing.

**Review by ammar92 — COMMENTED** [review]
Looks good to me. Just one remark that could improve it even more

**Review by underdarknl — APPROVED** [review]

**Pull request merged**